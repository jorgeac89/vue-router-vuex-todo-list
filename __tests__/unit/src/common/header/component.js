import { shallowMount } from '@vue/test-utils';
import Header from 'common/header';

describe('Header component', () => {
  it('renders correctly', () => {
    expect(shallowMount(Header)).toMatchSnapshot()
  });
});
