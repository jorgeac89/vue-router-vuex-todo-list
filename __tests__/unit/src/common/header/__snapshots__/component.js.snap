// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<div class="container-fluid bg-dark text-light mb-4">
  <div class="row align-items-center justify-content-around text-center">
    <div class="col-lg-6 text-lg-left">
      <h1 class="font-weight-light">Vue Router Vuex Todo List</h1>
      <p class="lead">
        Example of a web application developed with
        <strong>Vue.js</strong>,
        <strong>Vue Router</strong> and
        <strong>Vuex</strong>.
      </p>
    </div>
    <div class="col-lg-4 text-lg-right"><img alt="logo" src="undefined/images/logo.webp" class="img-fluid mb-3 my-lg-3"></div>
  </div>
</div>
`;
